src/components/Button.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+import buttonBG from '../assets/hc_button.svg'
+import { hexToRgba } from '../utility/utilities'
+
+const buttonSize = {
+  small: '60px',
+  default: '80px',
+  large: '300px',
+}
+
+const buttonColors = ['primary', 'secondary', 'tertiary']
+
+const StyledButton = styled.button`
+  display: block;
+  text-decoration: none;
+  font-family: 'HK Grotesk';
```

Comment:
Maybe we should add this to the themeprovider

Reply:
done!

Reply:
`font-family: ${p => p.theme.font};`